.circleci/regenerate.py

```diff
@@ -123,6 +123,14 @@ def unittest_workflows(indentation=6):
                 if device_type == 'gpu':
                     job['filters'] = gen_filter_branch_tree('master')
                 jobs.append({f"unittest_{os_type}_{device_type}": job})
+
+                if i == 0 and os_type == "linux" and device_type == "cpu":
```

Comment:
nit: this condition is quite specific,  and could just be put outside of the loop. the order of jobs doesn't matter either. either way is readable though, so let's leave it like this :)